easybuild/framework/easyblock.py

```diff
@@ -332,7 +337,7 @@ def fetch_patches(self, list_of_patches, extension=False, checksums=None):
                         patchspec['copy'] = suff
                     else:
                         patchspec['sourcepath'] = suff
-                if level:
+                if level is not None:
```

Comment:
It's only guessing, I don't think there's a way to fix it. Thanks for clarifying that setting the patch level to `0` revealed this issue.